include/engine/api/route_api.hpp

```diff
@@ -174,14 +174,13 @@ class RouteAPI : public BaseAPI
                 guidance::trimShortSegments(steps, leg_geometry);
                 leg.steps = guidance::postProcess(std::move(steps));
                 leg.steps = guidance::collapseTurnInstructions(std::move(leg.steps));
+                leg.steps = guidance::anticipateLaneChange(std::move(leg.steps));
```

Comment:
That was because of buildIntersection now removing use-lane. But with use-lane gone we can stay on the safe side and move it back down (perhaps). I am not 100% sure about how it affects the `can make all turns` combination below